src/App.tsx

```diff
@@ -416,6 +425,33 @@ const App = (): JSX.Element => {
     }
   }, []);
 
+  // Keeps fetching live data prices
+  const { error: pricesError } = useQuery(
+    PRICES_URL,
+    async () => {
+      const response = await fetch(PRICES_URL);
+      if (!response.ok) {
+        throw new Error('Network response for prices was not ok.');
+      }
+
+      const newPrices = await response.json();
+      // Update the store only if the price is actually changed
+      if (
+        newPrices.bitcoin.usd !== prices.bitcoin.usd ||
+        newPrices[RELAY_CHAIN_NAME].usd !== prices.collateralToken.usd ||
+        newPrices[BRIDGE_PARACHAIN_NAME].usd !== prices.governanceToken.usd
+      ) {
+        dispatch(updateOfPricesAction({
+          bitcoin: newPrices.bitcoin,
+          collateralToken: newPrices[RELAY_CHAIN_NAME],
+          governanceToken: newPrices[BRIDGE_PARACHAIN_NAME]
+        }));
+      }
+    },
```

Comment:
Can we move this fetcher into its own file in `src/services/fetchers/`?

Reply:
Ah, okay. Yes, I think it we can keep it for later refactoring, it's not a problem, was just a suggestion :)